src/interfaces/utils-interfaces/form-type.ts

```diff
@@ -4,3 +4,5 @@ export interface FormType extends HTMLFormElement {
   validate(): boolean
   resetValidation(): void
 }
+// To keep compatibility with the shared components that use FormIF and not FormType
+export interface FormIF extends FormType {}
```

Comment:
What would it take to change all "FormType" references to "FormIF" instead in this project?

Reply:
ok. I'll do it.